powershell-adapter/psDscAdapter/win_psDscAdapter.psm1

```diff
@@ -701,6 +704,20 @@ function GetClassBasedCapabilities {
     }
 }
 
+function Set-ValidPSModulePath {
+    [CmdletBinding()]
+    param()
+
+    end {
+        if (($env:PSModulePath -split [System.IO.Path]::PathSeparator) -contains '') {
```

Comment:
Seems like this could be simpler and faster just doing:

`$env:PSModulePath = [String]::Join([System.IO.Path]::PathSeparator, ($env:PSModulePath.Split([System.IO.Path]::PathSeparator, [System.StringSplitOptions]::RemoveEmptyEntries)))`